src/utility/index.js

```diff
@@ -199,18 +243,94 @@ const deleteYamlFile = yamlFileName => {
   });
 };
 
+/**
+ * Check if a file path ends in a valid json file name
+ *
+ * @param {string} fileName               - a json file name to validate
+ *
+ * @returns {boolean} isValidJsonFileName - true if file is a valid json file path
+ */
+const isValidJsonFileName = fileName => /.*\.json/.test(fileName);
```

Comment:
The regex really should all be moved into a single file and given variable names so it's not accidentally used incorrectly and so it can be tested.

Reply:
Do in another PR